container/Containerfile

```diff
@@ -0,0 +1,20 @@
+# Build stage
+
+FROM alpine AS build
+RUN apk add --no-cache --update build-base linux-headers python3 python3-dev py3-pip
+WORKDIR /build
+COPY ovirt-imageio.tar.gz .
+RUN python3 -m venv /venv
+RUN /venv/bin/pip install ovirt-imageio.tar.gz
+
+# Final stage
+
+FROM alpine
+RUN apk add python3
+RUN addgroup --system ovirtimg
+RUN adduser --system --ingroup ovirtimg ovirtimg
```

Comment:
We don't need to add the ovirtimg user and group since running as ovirtimg:ovirtimg
is not needed when running in a container and actually harmfull - when running as
ovirtimg we cannot access the mounted volumes since they are owned by root.